delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/model/search/SearchParams.scala

```diff
@@ -0,0 +1,65 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.model.search
+
+import ch.epfl.bluebrain.nexus.delta.sdk.model.{Label, ResourceRef}
+import org.apache.jena.iri.IRI
+
+sealed trait SearchParams {
+  def deprecated: Option[Boolean]
+  def rev: Option[Long]
+  def createdBy: Option[IRI]
+  def updatedBy: Option[IRI]
+  def types: Set[IRI]
+  def schemas: Set[ResourceRef]
+}
+
+object SearchParams {
+
+  /**
+    * Search parameters to filter realm resources.
+    *
+   * @param deprecated the optional deprecation status of the realm resources
+    * @param rev        the optional revision of the realm resources
+    * @param createdBy  the optional identity id who created the realm resource
+    * @param updatedBy  the optional identity id who updated the realm resource
+    * @param types      the collection of types of the realm resources
+    * @param schemas    the collection of schemas of the realm resources
+    */
+  final case class RealmSearchParams(
+      deprecated: Option[Boolean] = None,
+      rev: Option[Long] = None,
```

Comment:
I don't think rev makes sense... nor types and schemas, to be honest

Reply:
rev make sense, why not? types and schemas I can remove

Reply:
I don't see how: get all the realms at revision 2 would make any sense... :)